### PR TITLE
Sponsor page

### PR DESCRIPTION
**Pull request by Marcpezin**
The sponsor page includes a logo for each sponsor + a link to the website of the partners. 

Some partners didn't answered yet. 

Note: I wanted to remove the table border, It didn't work on my side. I don't know how to proceed with that. My best clue was that https://stackoverflow.com/questions/28806135/jekyll-kramdown-how-to-display-table-border

### CODE REVIEW

File: sponsor.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: "XCP-NG SPONSOR"
```
Comment (olivierlambert) [inline sponsor.md]: "XCP-ng sponsors"

File: sponsor.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+comments: false
+---
+XCP-ng sponsors span all industry sectors. They are trusting open source technologies to help them in their everyday business.
+
+## PLATINUM SPONSORS
```
Comment (olivierlambert) [inline sponsor.md]: IMHO, it's better to use titles in lower case: "Platinum sponsors"

File: sponsor.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+|[![](./assets/images/logos/kraemer-logo.png)](https://www.kraemer.de/)|[![](./assets/images/logos/axians-logo.png)](https://www.axians.de)|[![](./assets/images/logos/hostnet-logo.png)](https://www.hostnet.de/start.html)|
+|---------------------------------------|---------------------------------------|---------------------------------------|
+||||
+
+## GOLD SPONSORS
```
Comment (olivierlambert) [inline sponsor.md]: idem

File: sponsor.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+|[![](./assets/images/logos/kk-logo.png)](https://www.krankikom.de/?lang=en)|[![](./assets/images/logos/occentus-logo.png)](https://www.occentus.net/)|[![](./assets/images/logos/dd-wrt-logo.jpg)](https://www.dd-wrt.com/site/index)|
+|---------------------------------------|---------------------------------------|---------------------------------------|
+|[![](./assets/images/logos/exonet-logo.jpg)](https://www.exonet.nl/)|[![](./assets/images/logos/locaweb-logo.png)](www.locaweb.com.br)||
+
+## SILVER SPONSORS
```
Comment (olivierlambert) [inline sponsor.md]: idem

File: sponsor.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+|[![](./assets/images/logos/lawang-logo.png)](http://simdacloud.id/)|[![](./assets/images/logos/interconnect-logo.png)](https://www.interconnect.co.il/)|[![](./assets/images/logos/swisscenter-logo.png)](https://swisscenter.com)|
+|---------------------------------------|---------------------------------------|---------------------------------------|
+|[![](./assets/images/logos/microdata-logo.png)](https://microdata.co.uk)|||
+
+## SPONSORS
```
Comment (olivierlambert) [inline sponsor.md]: idem

**Review by olivierlambert — CHANGES_REQUESTED** [review]

File: sponsor.md (hunk 1 above)
Comment (Marcpezin) [reply sponsor.md]: corrected

File: sponsor.md (hunk 2 above)
Comment (Marcpezin) [reply sponsor.md]: ok - changed

File: sponsor.md (hunk 3 above)
Comment (Marcpezin) [reply sponsor.md]: ok - changed

File: sponsor.md (hunk 4 above)
Comment (Marcpezin) [reply sponsor.md]: ok - changed

File: sponsor.md (hunk 5 above)
Comment (Marcpezin) [reply sponsor.md]: ok - changed

**Review by olivierlambert — APPROVED** [review]

**Pull request merged**